Configuration.props

```diff
@@ -64,6 +64,7 @@
     <HOME Condition=" '$(HOME)' == '' ">$(USERPROFILE)</HOME>
     <AndroidPreviousFrameworkVersion Condition=" '$(AndroidPreviousFrameworkVersion)' == '' ">v1.0</AndroidPreviousFrameworkVersion>
     <AndroidToolchainCacheDirectory Condition=" '$(AndroidToolchainCacheDirectory)' == '' ">$(HOME)\android-archives</AndroidToolchainCacheDirectory>
+    <AndroidToolchainDirectory Condition=" '$(AndroidToolchainDirectory)' == '' And '$(RunningOnCI)' == 'true' And '$(HostOS)' == 'Darwin' ">$(HOME)\Library\Android</AndroidToolchainDirectory>
```

Comment:
If this line is specific to Azure DevOps  & Darwin, would it be better to put in YAML? you could just define `$(AndroidToolchainDirectory)`?

Reply:
@jonathanpeppers I don't know of a good way to ensure system variables expand when defined in a yaml variable, I think this may be "better" approach even though it's sub-optimal.